src/overlays/actors/ovl_Elf_Msg2/z_elf_msg2.c

```diff
@@ -1,4 +1,30 @@
+/*
+ * File: z_elf_msg2.c
+ * Overlay: ovl_Elf_Msg2
+ * Description: Targetable Navi check spot
+ *
+ * this.params
+ *     (p >> 8) & 0x3F          : Switch flag, set when actor is killed if (((p >> 8) & 0x3F) != 0x3F)
+ *                                (also see this.posRot.rot.y).
+ *     (p & 0xFF) + 0x100       : Message ID
+ *
+ * this.posRot.rot.x
+ *     (x > 0) && (x < 8))      : this->actor.unk_1F = (x - 1);
+ *
+ * this.posRot.rot.y
+ *     (y == -1)                : Actor is killed if room clear flag is set.
+ *     (y > 0x00) && (y < 0x41) : Actor is killed if switch flag (y - 1) is set.
+ *     (y > 0x40) && (y < 0x81) : Actor idles and is not interactable until switch flag (y - 0x41) is set, once
+ *                                running, actor is killed if switch flag ((p >> 8) & 0x3F) is set.
+ *     (y > 0x80)               : Actor idles forever and is not interactable (Bug?)
+ *     else:                    : Actor is killed if switch flag ((p >> 8) & 0x3F) is set.
+ *
+ * this.posRot.rot.z
+ *      (z != 1)                : Kill actor when closing the text box
```

Comment:
I wrote this up because the complicated flow made it hard for me to follow along with what these variables do in the code.

I know this won't be in final but I left it here so I could get some suggestions about where to put this kind of documentation. A lot of it can probably be sorted out using #defines, I'm interested to hear people's thoughts!